libs/barista-components/experimental/combobox/src/combobox.scss

```diff
@@ -41,7 +41,7 @@
 
   .dt-combobox-input,
   .dt-combobox-input::placeholder {
-    color: $disabledcolor;
+    @include dt-form-control-disabled-placeholder();
```

Comment:
I'm not entirely sure if we really need a mixin for this, but it's fine.

Reply:
At this moment we have at least 3 components which used it:
- input
- select
- combobox

so, decided to introduce 2 mixins, and according to https://github.com/dynatrace-oss/barista/issues/866#issuecomment-616968503

> Now every watermark in our components (Select, input field,...) is #898989 = Gray500, **however this color unfortunately does not yet follow our a11y guidelines**, but it would be more consistent.

I think that would be nice to change it in one place in the future ;)